core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -53,17 +54,45 @@ object Props extends Logger {
    * @param name key for the property to get
    * @return the value of the property if defined
    */
-  def get(name: String): Box[String] = Box(props.get(name))
+  def get(name: String): Box[String] = {
+    val tries:List[Box[Object]] =
+      prepended.map(m => Box.legacyNullTest(m get name)) ++
+      (props.get(name).toBox ::
+      appended.map(m => Box.legacyNullTest(m get name)))
+
+    tries.find(_.isDefined).getOrElse(Empty).map(interpolate)
+  }
+
+  private val INTERPOLATE = """(.*?)\Q${\E(.*?)\Q}\E([^$]*)""".r
+  private def interpolate(v:Object):String = {
```

Comment:
Good call on `INTERPOLATE`. I think originally I needed it to start with a capital to be a stable identifier for a pattern match.

As for `v`, I thought `interpolate` with a param `v` was sufficient to communicate that it is the value to interpolate. Since it is a private method, I didn't see any reason to err on the side of expounding further.  However, I'm accustomed to being told I'm wrong with my short variable names, so I've already updated it.